### ISSUE TITLE
BUG! about load_checkpoint() when i use retinanet_x101_64x4d_fpn_1x????

### ISSUE THREAD

**Issue by Peterisfar** [open-mmlab/mmdetection#640]
![image](https://user-images.githubusercontent.com/31852119/57631261-bfeb6300-75d1-11e9-8f47-e2668de387b6.png)

when i use retinanet_x101_64x4d_fpn_1x train result about epoch_1(example), get OSError.but when i use retinanet_r101_fpn_1x train result is OK, so where is the error ?
thanks

**Comment by hellock** [comment]
I cannot catch what you said...

**Comment by Peterisfar** [comment]
Sorry, what I described may not be clear. That is, when I normally call the weight file in the training step by step, such as epoch_18, config uses restinanet_x101_64x4d_fpn_1x, this weight loading problem occurs, but when I change the config to restinanet_r101_fpn_1x, and use the corresponding training weight such as epoch_18 (different from the above weights) , no such problem has arisen.

However, at the moment I tried to `pip install -e . `The problem is solved, but I don't know why.
